traffic_control/clients/python/trafficops/tosession.py

```diff
@@ -866,6 +866,17 @@ def assign_deliveryservice_servers_by_names(self, xml_id=None, data=None):
 		:raises: Union[LoginError, OperationError]
 		"""
 
+	@api_request(u'post', u'deliveryservice_requests', (u'1.1', u'1.2', u'1.3', u'1.4', u'1.5'))
```

Comment:
this should just be 2.0 the client is documented not to support any 1.x versions now

Reply:
Unsupported in aa00b978b0.